tools/build-osbs/build-osbs.sh

```diff
@@ -258,6 +258,9 @@ if [ -n "$OSBS_BUILD_USER" ]; then
     builduser="--build-osbs-user=$OSBS_BUILD_USER"
 fi
 
+# Enabling cekit on virtualenv
+source /home/jenkins/virtenvs/cekit/bin/activate
```

Comment:
Not sure if hard-coding the virtual environment location here is a good idea, since it will tie the script to the particular environment. On the other, if it is documented as a requirement for the script, that could be fine.

Reply:
Is there a env on Jenkins that we could use or configure to point to the virtualenv instead hardcode it here?

Reply:
can't we just do source ~/cekit/bin/activate ?